content/en/BestPractices/InputValidation.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Why It Is So Important To Validate User Input Safely
+linkTitle: why-safe-input-validation-is-important
+weight: 600
+toc_hide: true
+description: >
+  Input validation is a very important aspect of writing secure code. Consider all user input untrusted because attackers can easily manipulate it. If an application fails to properly validate untrusted user input, it might lead to security vulnerabilities such as Business Logic flaws, Denial of Service, Injections, and such. This article explores an attack scenario and walks you through the process of mitigating it. Also, learn the best practices to follow for safely validating user input.
+
+---
+
+{{% pageinfo %}}
+Every wondered why your application might be at a huge risk due to improper input validation? What are the best practices for securely validating user input? 
+{{% /pageinfo %}}
+
+## Description
+
+[Input validation](https://cwe.mitre.org/data/definitions/20.html) is a very important aspect of writing secure code. Consider all user input untrusted because attackers can easily manipulate it. If an application fails to properly validate untrusted user input, it might lead to security vulnerabilities such as [Business Logic flaws](https://owasp.org/www-community/vulnerabilities/Business_logic_vulnerability), [Denial of Service](https://en.wikipedia.org/wiki/Denial-of-service_attack), [Injection flaws](https://owasp.org/www-community/Injection_Flaws#:~:text=An%20injection%20flaw%20is%20a,connected%20to%20the%20vunlerable%20application.), and such. This article explores an abuse scenario and walks you through the process of mitigating it. Also, learn the best practices to follow for safely validating user input.
```

Comment:
The writer in me wants to suggest that we start off even stronger: 

`We recommend treating all user-supplied input as untrusted input, because it is possible for an attacker to manipulate that code. Input Validation is one of the best tools to prevent those manipulations. If an application does not properly validate untrusted input, it could lead to...`